src/APIFunctions/User.js

```diff
@@ -34,6 +34,19 @@ export async function updateLastLoginDate(email, token) {
 }
 
 
+export async function connectToDiscord(email, token) {
```

Comment:
wouldnt you want to put this into [`src/APIFunctions/Auth.js`](https://github.com/SCE-Development/Core-v4/blob/dev/src/APIFunctions/Auth.js)?

Reply:
will switch it over and see if if breaks anything